.github/workflows/update-dependencies.yml

```
@@ -0,0 +1,28 @@
+name: Update dependencies
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 6 * * 1'  # "At 06:00 on Monday."
```

```suggestion
    - cron: '0 6 * * 1'  # "At 18:00 on Sunday."
```
This would let me have some time to review (not that much probably)/merge this before the cibuildwheel update which is still at 06:00 on Monday and thus will have a bit more tests run before the business week really starts.